src/services/API/Question.ts

```diff
@@ -0,0 +1,34 @@
+import { apiClient } from "./RootClient";
+import { questionsEndpoint, getByQuestionSetEndpoint } from "./Endpoints";
+import { Question } from "../../types";
+
+export default {
+  /**
+   * returns the list of questions belonging to a question set
+   * @param {string} questionSetId - uuid of the question set
+   * @param {number | undefined} skip - skip a number of records
+   * @param {number | undefined} limit - limit the returned number of records
+   * @returns {Promise<QuizResponse>} a list of questions belonging to the given question set
+   */
+  async getQuestionsByQuestionSet(
```

Comment:
recommend keeping this consistent across frontend and backend
```suggestion
  async getQuestions(
```